config/flags_node.go

```diff
@@ -171,6 +171,12 @@ var (
 		Usage: "IP address to bind API to",
 		Value: "127.0.0.1",
 	}
+	// FlagTequilapiAllowedHostnames Restrict hostnames in requests' Host header to following domains.
+	FlagTequilapiAllowedHostnames = cli.StringFlag{
```

Comment:
Average person doesn't run node CLI!

There are two options:

* We expose this parameter visible and it just adds another line in help output.
* We hide it and whoever uses it will have to open source code to enable remote access by hostname to TequilAPI, otherwise receiving 403 for unknown reason.

I picked second option.